aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,361 @@
-More content coming soon.
+## Messaging Voice, Tone, and Content
```

Comment:
Wondering if we should provide any sort of Table of Contents at the top of this page to make it easier to navigate/discover the available topics?

Reply:
I would second this as being a good idea

Reply:
```suggestion
Messaging within HPE applications should be informative, concise, and trustworthy. 
It should be delivered in active, positive, objective, conversational 
American English and written from a third-person [point of view](#point-of-view).
Executed properly, messaging within HPE applications should leave users 
feeling and perceiving HPE and its products to be intelligent, experienced, 
inclusive, and non-threatening. 

## Messaging Voice, Tone, and Content
```